examples/simple/src/main.rs

```diff
@@ -27,149 +28,156 @@ enum RenderState<'s> {
     Suspended(Option<Arc<Window>>),
 }
 
-fn main() -> Result<()> {
-    // Setup a bunch of state:
-
-    // The vello RenderContext which is a global context that lasts for the lifetime of the application
-    let mut render_cx = RenderContext::new();
+struct RenderStateWithContext<'s> {
```

Comment:
I think we should rename this to something which describes what it is for, rather than what it contains. E.g. `SimpleVelloApp`?

Reply:
Thanks for the suggestion! `SimpleVelloApp` sounds better.